blocks/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+
+# Create your models here.
+
+
+class Block(models.Model):
+    '''
+    This represents a block in the blockchain
+    '''
+    hash = models.CharField(max_length=40, editable=False)
```

Comment:
This will work the way it is, but just sharing some insights. When I wrote my blockchain protocol, it was in a flask with txt files as databases (We had to use a universal file system that we can distribute across a variety of OS). Naturally, there was an issue with the security of hashes being in their original form, anyone could guess the hashing format from their length of character. Instead, we saved a Positive Integer value and used getters/setters to convert it into hashing algorithms.

Also, there is a HexaDecimal field that does the same thing in Django. https://pypi.org/project/DjangoHexadecimalField/